docs/input/docs/ci-systems/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+Order: 10
+Title: Overview
+Description: On using one or more CI-systems
+---
+
+## Using a CI-System
+
+Generally using Cake.Recipe on any ci-system is not different from using it locally. 
+However, some CI-systems need different preparations than others.
+
+## Using multiple CI-systems (or running multiple builds on the same system)
+
+When a single commit is built multiple times 
+(because of multiple CI-systems and/or multiple builds in the same system)
+some tasks (generelly all tasks that involve some publishing) should only run once (to avoid publishing more than once).
+
+To ensure that those tasks are run only once, Cake.Recipe provides two setting:
+
+* [preferredBuildProviderType](../fundamentals/set-parameters#preferredbuildprovidertype)
+* [preferredBuildAgentOperatingSystem](../fundamentals/set-parameters#preferredbuildprovidertype)
```

Comment:
I believe the style used in the markdown already in the repository is using `-` as the unordered list denotation.

```suggestion
- [preferredBuildProviderType](../fundamentals/set-parameters#preferredbuildprovidertype)
- [preferredBuildAgentOperatingSystem](../fundamentals/set-parameters#preferredbuildprovidertype)
```